rpcd/playbooks/roles/elasticsearch/tasks/elasticsearch_install.yml

```diff
@@ -31,7 +31,9 @@
   pip:
     name: "{{ item }}"
     state: latest
-    extra_args: "{{ pip_install_options | default('') }}"
+    extra_args: >-
+      {{ (pip_install_upper_constraints is defined) | ternary('--constraint ' + pip_install_upper_constraints | default(''),'') }}
```

Comment:
pip install is a dependency indeed.

For the extra safety, we could have the group vars all in rpc, but I will have a commit soon to remove the need for it.
In the meantime, we can follow what @odyssey4me said.